deposit-app/src/main/webapp/WEB-INF/deposit-jobs-context.xml

```diff
@@ -191,6 +191,7 @@
         <property name="clamClient" ref="clamClient" />
         <property name="maxQueuedJobs" value="${job.fileValidation.maxQueuedJobs:5}" />
         <property name="executorService" ref="fileValidationExecutor" />
+        <property name="maxStreamSize" value="${clamd.maxStreamSize:64000000}" />
```

Comment:
what is this for?

Reply:
that's what sets the maxStreamSize. I copy and pasted its value from the clam client. It pulls from the clamd.maxStreamSize variable is set, and falls back to 64mb if not